components/Releases/CreateRelease.jsx

```diff
@@ -364,6 +379,23 @@ export default function CreateRelease({
             is usually <code>your-name.bandcamp.com/yum</code>.
           </small>
 
+          {profileData.is_subscribed || profileData.dlcm_friend ? (
+            <div className="input-wrapper">
+              <label className="label" htmlFor="playerEmbed">
+                Bandcamp audio player embed
+              </label>
+              <PopoverTip
+                message={`Paste a bandcamp embed iframe here. We set what the player will look like for consistency, so it doesn't matter what size or customizations you make.`}
```

Comment:
- "Paste a Bandcamp HTML embed " might be more consistent with what the user will see in Bandcamp. It should also keep them from accidentally choosing the wordpress shortcode instead.
- What are the steps a user needs to take to get the embed code? What if they try to set other customizations below the embed code?

<img width="300" alt="image" src="https://github.com/frankmarra/DLCM/assets/1704038/ab949879-0759-45df-bce6-7c775b3562f2">

Reply:
Changed the wording. There are a bunch of steps. Too many to put in this popover tip. It doesnt matter what the user chooses as far as customizations. We only grab the album number and add that to a hard coded iframe.